src/app/card/basic-card/example/card-basic-example.component.html

```diff
@@ -5,7 +5,7 @@
         <pfng-card [config]="config"
                    (onActionSelect)="handleActionSelect($event)"
                    (onFilterSelect)="handleFilterSelect($event)">
-          <pfng-chart-sparkline [config]="chartConfig" [chartData]="chartData" ></pfng-chart-sparkline>
+          <pfng-sparkline-chart [config]="chartConfig" [chartData]="chartData" ></pfng-sparkline-chart>
```

Comment:
Isn't this a Major breaking change?

Reply:
No, it's a deprecation. This matches the other component tags, but both will be available.